prisma/migrations/20210529200211_add_tables_for_nextjs_auth/migration.sql

```diff
@@ -0,0 +1,77 @@
+-- CreateTable
+CREATE TABLE "Account" (
+    "id" TEXT NOT NULL,
+    "userId" TEXT NOT NULL,
+    "providerType" TEXT NOT NULL,
+    "providerId" TEXT NOT NULL,
+    "providerAccountId" TEXT NOT NULL,
+    "refreshToken" TEXT,
+    "accessToken" TEXT,
+    "accessTokenExpires" TIMESTAMP(3),
+    "createdAt" TIMESTAMP(3) NOT NULL DEFAULT CURRENT_TIMESTAMP,
+    "updatedAt" TIMESTAMP(3) NOT NULL,
+
+    PRIMARY KEY ("id")
+);
+
+-- CreateTable
```

Comment:
For comments, I think having the same comment isn't too useful since the actual line already says CREATE TABLE. I think it would be beneficial to say the specific table you're creating

Reply:
Noted - but since we don't want migrations right now this doesn't matter. Will keep this in mind for the future.